docs/notebooks/example1-with-sensors.xml

```diff
@@ -223,6 +223,27 @@
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-11"/>
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-12"/>
                   </AnnualFuelUseLinkedTimeSeriesIDs>
+                  <Emissions>
+                    <Emission>
+                      <EmissionBoundary>Indirect</EmissionBoundary>
+                      <EmissionsType>CO2e</EmissionsType>
```

Comment:
This needs to be changed to Greenhouse Gas Emissions

Reply:
@laurenliz22 I had a second thought about this - according to Wikipedia "greenhouse gas" includes `CO2`, `CH4`, `N2O`, `O3` and some others, which are partially covered by the enumerations under `EmissionsType`. My understanding is that `EmissionsType` element specifies what type of greenhouse gas the emissions primarily include, so 1. we might don't want to add `MtCO2e` here (the notebook assumes we do), and 2. the intention of existing `CO2e` might be "non-specified" or "combinations" of greenhouse gas emissions apart from other specified enums, so we could either leave it as it is or rename it to `Greenhouse gas emissions` as you suggest here.